src/config_format/flb_cf_yaml.c

```diff
@@ -453,7 +566,7 @@ static int consume_event(struct flb_cf *cf, struct local_ctx *ctx,
     case STATE_CUSTOM:
         switch (event->type) {
         case YAML_SEQUENCE_START_EVENT:
-            s->state = STATE_CUSTOM_SEQUENCE;
+            s->state = STATE_PLUGIN_TYPE;
```

Comment:
'customs' section is for custom plugin.
So I set `STATE_PLUGIN_TYPE` like other plugins.